scripts/util/plot.py

```diff
@@ -52,30 +52,35 @@ def output_graph(ref_param, result_param, output_dir, save_param, op_param):
     fig_2d_trj = plt.figure("2D_Trajectory", figsize=(16, 9), dpi=120)
     ax_2d_trj = fig_2d_trj.add_subplot(111)
     ax_2d_trj.set_title("2D Trajectory", fontsize=save_param.title_font_size)
-    ax_2d_trj.scatter(ref_param.df["x"], ref_param.df["y"], c="k", label="Reference")
-    ax_2d_trj.scatter(result_param.df["x"], result_param.df["y"], c="r", s=2, label="Result")
-    ax_2d_trj.plot([ref_param.df["x"], result_param.df["x"]], [ref_param.df["y"], result_param.df["y"]], "g-", linewidth=0.2, zorder=1)
-    if op_param.display_ellipse == True:
-        for i in range(len(result_param.df)):
-            e = patches.Ellipse(xy=(result_param.df['x'][i],result_param.df['y'][i]), width=result_param.df["ellipse_long"][i]*2, height=result_param.df["ellipse_short"][i]*2, angle=math.degrees(result_param.df["ellipse_yaw"][i]), alpha=0.3,color='m') 
-            ax_2d_trj.add_patch(e)
-
-    if save_param.progress_info != 0:
-        c = 1
-        for i in range(1, len(result_param.df)):
-            if save_param.progress_info == 1 and i >= save_param.interval * c: # number
-                ax_2d_trj.text(result_param.df["x"][i], result_param.df["y"][i], i, va='bottom')
-                c += 1
-            elif save_param.progress_info == 2 and time[i] >= save_param.interval * c: # time
-                ax_2d_trj.text(result_param.df["x"][i], result_param.df["y"][i], round(time[i],3), va='bottom')
-                c += 1
-            elif save_param.progress_info == 3 and time[i] >= save_param.interval * c: #rostime
-                ax_2d_trj.text(result_param.df["x"][i], result_param.df["y"][i], ref_param.df["bottom"][i], va='bottom')
-                c += 1
-            elif save_param.progress_info == 4 and distance[i] >= save_param.interval * c: # distance
-                ax_2d_trj.text(result_param.df["x"][i], result_param.df["y"][i], round(distance[i],1), va='bottom')
-                c += 1
-            
+    for param in [ref_param] + result_params:
+        param.df_temp, param.df = param.df, param.df.iloc[::10] # dilute data by 10
```

Comment:
This is a good process, so could you fix that user can set the number of dilute data in yaml file? 
Like below. Then add it to sample yaml file also.
```

## Display
# Horizontal axis
axis_type: 0 # [0]:time, [1]:distance

# Trajectory graph dilution
dilute data: 0

# Trajectory graph numbering
progress_info: 0 # [0]:no display, [1]: number, [2]:time, [3]:ros time, [4]:distance
interval: 0 # progress_info is [2]:per second, [3]:per second, [4]:per meter
```